### PR TITLE
CNV-24175-B: Check UI overview assembly against 4.13 web console

### PR DESCRIPTION
**Pull request by danielclowers**
**Version(s):**
4.13+

**Issue:**
https://issues.redhat.com/browse/CNV-24175

**Link to docs preview:**
https://59375--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-web-console-overview.html

**QE review:**
- [x] QE has approved this change.

**Additional information:**
NA

### CODE REVIEW

**Review by mjpytlak — COMMENTED** [review]
@danielclowers peer review lgtm.

**Review by maxwelldb — COMMENTED** [review]
Noticed something to change. Relabel when ready. 👍

File: virt/virt-web-console-overview.adoc [hunk 1]
```diff
@@ -919,7 +930,9 @@ You can create snapshots and restore VirtualMachines from snapshots on the *Snap
 |*Snapshot* table
 |List of snapshots
 
-Click the Options menu {kebab} beside a snapshot to select *Edit labels*, *Edit annotations*, *Edit VirtualMachineSnapshot*, *Delete VirtualMachineSnapshot*.
+Click the snapshot name to edit the labels or annotations.
+
+Click the Options menu {kebab} beside a snapshot to select *Restore* or *Delete*.
```
[the remark this answers is not in the document]
Comment (danielclowers) [reply virt/virt-web-console-overview.adoc]: Thanks! I found quite a few instances of this elsewhere in the docs so I went ahead and fixed those too. Once the new check completes successfully I'll relabel.

**Review by skrthomas — COMMENTED** [review]
LGTM :) nice work!

File: virt/virt-web-console-overview.adoc [hunk 2]
```diff
@@ -1384,7 +1409,7 @@ You can manage MigrationPolicies for your workloads on the MigrationPolicies pag
 |*MigrationPolicies* table
 |List of MigrationPolicies
 
-Click the Options menu {kebab} beside a MigrationPolicy to select *Edit* or *Delete*.
+Click the *Options* menu {kebab} beside a MigrationPolicy to select *Edit* or *Delete*.
```
Comment (apinnick) [inline virt/virt-web-console-overview.adoc:1412]: Actually, we do not mark a menu name unless it is actually indicted with a tooltip or similar.

Comment (apinnick) [reply virt/virt-web-console-overview.adoc:1412]: Yes, I found those sections as well. I asked Andrea Hoffer to confirm.

Comment (apinnick) [reply virt/virt-web-console-overview.adoc:1412]: In the meantime, no need to change. I will resolve this conversation.

**Pull request merged**